web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java

```diff
@@ -9,17 +9,38 @@
  */
 package org.openmrs.web.filter.update;
 
-import liquibase.changelog.ChangeSet;
-import liquibase.exception.LockException;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
```

Comment:
Can you avoid changing these imports?

Reply:
I think it is mvn formating that may have caused the changes in the imports.

Reply:
Are you sure?

Reply:
These imports were due to automated formatting tests.

Reply:
I am closing this PR to resolve the unwanted imports